### PR TITLE
Add maintainer: tehsphinx

### PR DESCRIPTION
**Pull request by tehsphinx**

### CODE REVIEW

File: config/maintainers.json [hunk 1]
```diff
@@ -90,6 +90,16 @@
       "link_url": "https://www.sborza.com/",
       "avatar_url": null,
       "bio": "I'm a Canadian (based in Chicago) golang, C and python developer working on projects within the gaming, finance and data-science worlds respectively. My favorite freetime task is coding challenges and I'm more than happy to help people talk through solutions!"
+    },
+    {
+      "github_username": "tehsphinx",
+      "alumnus": true,
```
Comment (iHiD) [inline config/maintainers.json]: I think this should be `false` unless there's bad news you've not told me!

Comment (leenipper) [reply config/maintainers.json]: Yes, anyone intending to be active is not an alum.  Good catch @iHiD .  And @sebito91 I missed this property on yours, so you will want to correct it.

**Comment by tehsphinx** [comment]
@iHiD fixed :)

I guess I did not properly read the translations when looking it up. :D

**Comment by iHiD** [comment]
Phew!!

**Pull request merged**